compiler/luci-interpreter/src/kernels/TestUtils.h

```diff
@@ -58,6 +60,113 @@ template <typename T> std::vector<T> extractTensorData(const Tensor &tensor)
 std::vector<::testing::Matcher<float>> ArrayFloatNear(const std::vector<float> &values,
                                                       float max_abs_error = 1.0e-5f);
 
+template <typename T>
+inline std::vector<T> quantize(const std::vector<float> &data, float scale, int32_t zero_point)
+{
+  std::vector<T> q;
+  for (const auto &f : data)
+  {
+    if (getElementType<T>() == DataType::FLOAT32)
```

Comment:
How about removing this if statement and adding a below assertion at the beginning of this function? It seems that T should not be `float`.

E.g., `assert(getElementType<T>() != DataType::FLOAT32);`

Reply:
Okay, I fix like what you suggested.